.github/workflows/versioning.yml

```
@@ -45,8 +45,9 @@ jobs:
       - name: Commit and branch updated version
         uses: peter-evans/create-pull-request@v4
         with:
-          title: "v${{ env.PACKAGE_VERSION }}"
+          title: "[Version update]"
```

i think it's actually nice to have the version in the title :-) i just meant it shouldn't be the source of truth
```suggestion
          title: "v${{ env.PACKAGE_VERSION }}"
```